metatensor-core/CMakeLists.txt

```diff
@@ -69,18 +69,16 @@ set(EXTRA_RUST_FLAGS "" CACHE STRING "Flags used to build rust code")
 set(CMAKE_MACOSX_RPATH ON)
 set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}")
 
-if (${METATENSOR_MAIN_PROJECT})
-    if("${CMAKE_BUILD_TYPE}" STREQUAL "" AND "${CMAKE_CONFIGURATION_TYPES}" STREQUAL "")
-        message(STATUS "Setting build type to 'release' as none was specified.")
-        set(CMAKE_BUILD_TYPE "release"
-            CACHE STRING
-            "Choose the type of build, options are: debug or release"
-        FORCE)
-        set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS release debug)
-    endif()
+if("${CMAKE_BUILD_TYPE}" STREQUAL "" AND "${CMAKE_CONFIGURATION_TYPES}" STREQUAL "")
+    message(STATUS "Setting build type to 'release' as none was specified.")
```

Comment:
Should release really the default?

Reply:
What else would you want to see? I think we need to build with optimizations by default so random users get the full performance of the code. We could also include some basic debug information, but we don't really make use of it anywhere.

Reply:
Yeah most likely optimizing everything if nothing is given is a good idea.